pypika/terms.py

```diff
@@ -84,8 +84,13 @@ def between(self, lower, upper):
 
     def isin(self, arg):
         if isinstance(arg, (list, tuple, set)):
-            return ContainsCriterion(self, ListField([self._wrap(value) for value in arg]))
-        return ContainsCriterion(self, arg)
+            return ContainsCriterion(self, ListField([self._wrap(value) for value in arg]), isin=True)
+        return ContainsCriterion(self, arg, isin=True)
+
+    def isnotin(self, arg):
```

Comment:
@twheys There is no __not__() method for object instances; only the interpreter core defines this operation (see https://docs.python.org/3/library/operator.html?highlight=__not__). I even tried, but custom __not__ methods are never executed.